gatherers/theme-color.js

```diff
@@ -20,15 +20,15 @@ const Gather = require('./gather');
 
 class ThemeColor extends Gather {
 
-  static gather(options) {
+  afterPageLoad(options) {
     const driver = options.driver;
 
     return driver.querySelector('head meta[name="theme-color"]')
       .then(node => node && node.getAttribute('content'))
       .then(themeColorMeta => {
-        return {themeColorMeta};
+        this.artifact = {themeColorMeta};
       });
   }
 }
 
-module.exports = ThemeColor;
+module.exports = new ThemeColor();
```

Comment:
in general I really dislike exporting a singleton instance, it can tend to make things harder to stub and interact with.

This will be cleaned up in the future though I suppose the moment we switch to flat objects